### PR TITLE
Added docs for snap package module

### PR DESCRIPTION
**Pull request by nickanderson**
Ticket: CFE-2811
Changelog: Title

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

File: reference/functions/getenv.markdown [hunk 1]
```diff
@@ -23,7 +23,7 @@ variable you are querying.
 
 Output:
 
-[%CFEngine_include_snippet(getenv.cf, #\+begin_src\s+example_output\s*, .*end_src)%]
+[gg%CFEngine_include_snippet(getenv.cf, #\+begin_src\s+example_output\s*, .*end_src)%]
```
Comment (craigcomstock) [inline reference/functions/getenv.markdown]: what's the `gg` for ?

Comment (nickanderson) [reply reference/functions/getenv.markdown]: for correction!

Comment (nickanderson) [reply reference/functions/getenv.markdown]: fixed, squashed.

**Pull request merged**